design/ceph/ceph-external-cluster.md

```diff
@@ -157,3 +157,41 @@ monitoring:
 4. Create a new metric Service
 
 5. Create an Endpoint resource based out of the IP addresses either discovered or provided by the user in the script
+
+## Steps for creating external rook-ceph cluster
+
+1. Run the python script [create-external-cluster-resources.py](/deploy/examples/create-external-cluster-resources.py) on a standalone [ceph cluster](https://docs.ceph.com/en/pacific/cephadm/#cephadm) for creating all users/keys,
```

Comment:
I'd suggest having two different level-3 heading sections, one for commands to run in the source cluster, and for the target cluster. It's confusing if they're all in a single section.